tutorial/Lab1/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1beta2
```

Comment:
I may have missed it, but where is this file (and service.yaml) used?

Reply:
Redundant files and now removed